### PR TITLE
Minor fixes on Rumor

### PR DESCRIPTION
**Pull request by cortze**
Fixed:
- Missing `connectall` info on the `peer --help` flag.
- Wrong method to show PeerId of the incoming message on gossip topics.
- Fixed bug that was making `gossip list` just showing one of the topics.

### CODE REVIEW

File: control/actor/gossip/list.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ func (c *GossipListCmd) Run(ctx context.Context, args ...string) error {
 	topics := make([]string, 0)
 	c.GossipState.Topics.Range(func(key, value interface{}) bool {
 		topics = append(topics, key.(string))
-		return false
+		return true
```
Comment (protolambda) [inline control/actor/gossip/list.go:24]: good catch

File: control/actor/gossip/log.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/golang/snappy"
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
 	"github.com/protolambda/rumor/control/actor/base"
-	"github.com/sirupsen/logrus"
+    "github.com/sirupsen/logrus"
```
Comment (protolambda) [inline control/actor/gossip/log.go:10]: hmm, some weird formatting change, will run `go fmt` later to fix

File: control/actor/gossip/log.go [hunk 3]
```diff
@@ -55,7 +55,7 @@ func (c *GossipLogCmd) Run(ctx context.Context, args ...string) error {
 						msgData = msg.Data
 					}
 					c.Log.WithFields(logrus.Fields{
-						"from":      msg.GetFrom().String(),
+						"from":      msg.ReceivedFrom.String(),
```
Comment (protolambda) [inline control/actor/gossip/log.go:58]: Good catch, the difference is:

`msg.GetFrom()` returns the `msg.Message.From`: whatever the gossip message claims as origin. In Eth2 this field is unused, and along with signatures etc. completely rejected (due to signature policy). Eth2 is content-addressed, and not origin-stamped.

`msg.ReceivedFrom`: the peer we first learned the message from, may not be the original publisher of the message.

**Review by protolambda — APPROVED** [review]
LGTM, thanks!

**Review by etoja07 — APPROVED** [review]

**Pull request merged**